LeagueCommissioner/src/main/java/com/revature/daos/TeamDAOImpl.java

```diff
@@ -1,5 +1,31 @@
 package com.revature.daos;
 
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+import com.revature.beans.Team;
+import com.revature.util.HibernateUtil;
+
 public class TeamDAOImpl implements TeamDAO {
 
+	@Override
+	public void createTeam(Team t) {
+		Session session= HibernateUtil.getSession();
+		Transaction tx = null;
+		
+		try {
+			tx = session.beginTransaction();
+			session.persist(t); //This method persists the object

```

Comment:
since we are creating a Team for the first time, we can just save it
we should probably have persist and rollback when we update records